diagnostic.md

```diff
@@ -100,11 +135,28 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH http://www.example.com/donuts/42 \
+--header "Content-Type: application/json" \
+--data '{
+  "name": "Krüller"
+}'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+const kruller = {
+  "name": "Krüller"
+};
+
+kruller = JSON.stringify(kruller);
+
+let changeDonut = function (id, donut) {
+  return $.ajax({
+    url: 'http://www.example.com/donuts' + id,
+    method: 'PATCH',
+    data: donut,
+});
+
+changeDonut(42, kruller);
```

Comment:
This is a different approach, but it works. Would you want to hard code, all of these data values and stringify them for every HTTP verb? Think about a more dry to do these requests--for instance, would you want to set up these variables for every single request in your tic tac toe game?